src/lib/features/instance-stats/getLicensedUsers.ts

```diff
@@ -0,0 +1,28 @@
+import type { Db } from '../../server-impl';
+
+export type GetLicensedUsers = () => Promise<number>;
+
+export const createGetLicensedUsers =
+    (db: Db): GetLicensedUsers =>
+    async () => {
+        const result = await db('users')
+            .countDistinct('email_hash as activeCount')
+            .whereNotNull('email_hash')
+            .andWhere(function () {
+                this.whereNull('deleted_at').orWhere(
+                    'deleted_at',
+                    '>=',
+                    db.raw("NOW() - INTERVAL '30 days'"),
```

Comment:
We do not know yet, we had discussion on the project channel and last 30 days seems to be the best. Customers are invoices on any day from 1 to 30, so it is hard to define a strict cycle.

Reply:
Right, got it. But my thinking here (and tell me if I'm wrong) is that if you're invoiced on a set date -- say the 30th -- then this will miss a day every other month. E.g., if we take the sample on the 30th of November, then that'll give you for all of November. But we also took it on the 30th of October. So what happens to October 31st? 🎃 

Sorry for the late reply, by the way. Turns out I'm not getting notifications for some of these. Probably because the PR was merged 🤔